SymPy tests fail in IPython 5b3

Run 

```
./bin/test ipython
```

from the SymPy repo with the IPython 5b3 beta installed. This is the test failure.

```
______________________________________________________________________________________________________________________________________________________________
_________________________________________ sympy/interactive/tests/test_ipythonprinting.py:test_print_builtin_option __________________________________________
  File "/users/aaronmeurer/documents/python/sympy/sympy/sympy/interactive/tests/test_ipythonprinting.py", line 66, in test_print_builtin_option
    raises(KeyError, lambda: app.user_ns['a'][0]['text/latex'])
  File "/Users/aaronmeurer/Documents/Python/sympy/sympy/sympy/utilities/pytest.py", line 81, in raises
    raise AssertionError("DID NOT RAISE")
AssertionError: DID NOT RAISE
```

--- a/IPython/terminal/ptshell.py
+++ b/IPython/terminal/ptshell.py
@@ -162,6 +162,11 @@ class TerminalInteractiveShell(InteractiveShell):
         else:
             toggle_set_term_title(False)
 
+    def init_display_formatter(self):
+        super(TerminalInteractiveShell, self).init_display_formatter()
+        # terminal only supports plain text
+        self.display_formatter.active_types = ['text/plain']
+
     def init_prompt_toolkit_cli(self):
         self._app = None
         if self.simple_prompt:

--- a/IPython/terminal/tests/test_interactivshell.py
+++ b/IPython/terminal/tests/test_interactivshell.py
@@ -95,6 +95,11 @@ class InteractiveShellTestCase(unittest.TestCase):
             ip.input_splitter.python_line_transforms.remove(transformer)
             ip.input_transformer_manager.python_line_transforms.remove(transformer)
 
+    def test_plain_text_only(self):
+        ip = get_ipython()
+        formatter = ip.display_formatter
+        assert formatter.active_types == ['text/plain']
+
 
 class SyntaxErrorTransformer(InputTransformer):
     def push(self, line):

